python/keepsake/pl_callback.py

```diff
@@ -56,15 +57,18 @@ def __init__(
         """
 
         super().__init__()
-        self.filepath = filepath
+        self.filepath = Path(filepath).resolve()
         self.params = params
         self.primary_metric = primary_metric
         self.period = period
         self.save_weights_only = save_weights_only
         self.last_global_step_saved = -1
 
     def on_pretrain_routine_start(self, trainer, pl_module):
-        self.experiment = keepsake.init(path=".", params=self.params)
+        self.experiment = keepsake.init(
+            path=str(self.filepath.parent),
```

Comment:
The experiment path and the checkpoint path should probably be two different paths, since in the normal case you want to save your working directory once at `init()` time and each version of a checkpoint at `experiment.checkpoint()` time. Can we add an `experiment_path` argument to `__init__` and use that here?

Reply:
Ah oops. This was a misunderstanding of how the `init` function operated. I'll fix that and update.